respect-lib-shared/src/androidMain/kotlin/world/respect/shared/domain/biometric/BiometricAuthUseCaseAndroidImpl.kt

```diff
@@ -0,0 +1,12 @@
+package world.respect.shared.domain.biometric
+
+class BiometricAuthUseCaseAndroidImpl(
```

Comment:
There is no need to have a separate BiometricAuthUseCaseAndroidImpl _and_ ChannelHost. As per the credential use cases, the Android implementation can host the channel.